ext/js/app/frontend.js

```diff
@@ -919,12 +927,13 @@ export class Frontend {
 
     /**
      * @param {boolean} allowEmptyRange
+     * @param {boolean} lazy
```

Comment:
Can you add a comment on what lazy does? Usually it's in the context of lazy evals but this is not the case it seems

Reply:
Removed `lazy` and put `disallowExpandSelection` instead. Should be more clear what that does.